tests/test_ec2_metadata.py

```diff
@@ -61,6 +61,34 @@ def test_account_id(em_requests_mock):
     assert ec2_metadata.account_id == "1234"
 
 
+def test_tags(em_requests_mock):
```

Comment:
just like the attributes, the tests were in alphabetical order based on the attribute names - rearranging